src/pretix/control/views/event.py

```diff
@@ -1297,6 +1297,11 @@ def form_valid(self, form):
                     form.cleaned_data['payment_banktransfer_%s' % f]
                 )
 
+            self.request.event.settings.set(
+                'payment_banktransfer_code_prefix',
+                form.cleaned_data['payment_banktransfer_code_prefix']
+            )
```

Comment:
I think you can remove this completely, the prefix will not become part of the quick-setup form.